packages/core/types/tests.tsx

```diff
@@ -123,3 +124,62 @@ interface TestTheme1 {
     )
   }}
 </ClassNames>
+
+// issue 1532 https://github.com/emotion-js/emotion/pull/1532
+interface HaveObjectCss<T = any> {
+  less: ObjectInterpolation<T>
+}
+// prior to 1532 the css prop being passed less would fail
+const LessComponent = <T extends any>({ less }: HaveObjectCss<T>) => (
+  <div css={less}>I have Css</div>
+)
+
+interface AnimationTheme {
+  from: { top: number }
+  to: { top: number }
+}
+
+// BEGIN NON TEMPLATE ANIMATIONS
+// without restricting less
+;<LessComponent
+  less={keyframes<AnimationTheme>({ from: { top: 0 }, to: { top: 20 } })}
```

Comment:
I don't see how `AnimationTheme` is relevant here, `keyframes`' both inputs and return value are not dependant on it at all. 

Maybe the fix you are after here is somewhere else? Might be that `args` passed to `keyframes` are wrongly types and shouldn't use `Interpolation` type at all. I'm not 100% sure if this is the correct answer - this is the direction I would investigate right now, because parametrizing `keyframes` with `Theme` doesn't make much sense to me.